ggshield.yaml

```diff
@@ -1,12 +1,14 @@
 package:
   name: ggshield
   version: 1.30.2
-  epoch: 0
+  epoch: 1
   description: Find and fix 360+ types of hardcoded secrets and 70+ types of infrastructure-as-code misconfigurations.
   copyright:
     - license: MIT
   dependencies:
     runtime:
+      - git
```

Comment:
@joshrwolf looks like new runtime deps were introduced here - wad this intentional? Any context?

Reply:
the tests I added failed unless `git` was installed, so I added it to the runtime